crowbar_framework/lib/crowbar/upgrade.rb

```diff
@@ -45,11 +45,13 @@ def supported?
     protected
 
     def knife_files
+      logger.debug "Upgrading chef backup files"
       @data.join("knife", "databags", "barclamps").rmtree
       knife_path = @data.join("knife")
 
       crowbar_databags_path = knife_path.join("databags", "crowbar")
       crowbar_databags_path.children.each do |file|
+        logger.debug "Upgrading #{file.to_s}"
```

Comment:
Lint/StringConversionInInterpolation: Redundant use of `Object#to_s` in interpolation. (https://github.com/bbatsov/ruby-style-guide#no-to-s)